Makefile

```diff
@@ -72,10 +72,10 @@ html-ui-bindata: html-ui $(BINDATA_TOOL)
 html-ui-bindata-fallback: $(BINDATA_TOOL)
 	(cd internal/server && $(BINDATA_TOOL) -fs -tags !embedhtml -o "$(CURDIR)/internal/server/htmlui_fallback.go" -pkg server index.html)
 
-travis-release: test-with-coverage lint vet verify-release integration-tests upload-coverage website stress-test
+travis-release: test-with-coverage lint vet verify-release integration-tests robustness-tests upload-coverage website stress-test
```

Comment:
Not sure whether or not it should be included in travis-release.
- In principle, yes, we want to make sure that the robustness tests pass before releasing.
- OTOH, IIRC, we don't (yet) exercise `travis-release` during CI. We could do something custom for the kastenhq repo to test this out before sending it upstream.

Reply:
Just looked at the travis output, and it seems that `travis-release` is executed.
OK, I have not carefully looked at the Makefile and the travis config then.